data/features.ts

```diff
@@ -2,4 +2,5 @@ export const FEATURES = {
   blogs: true,
   testimonials: true,
   subscribe: false,
+  product: false,
```

Comment:
Can you confirm? I think we want to add this as a feature so it should be true. I am not sure though